crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -449,7 +449,7 @@ fn dependency_excludes_range_of_compatible_versions() {
       × No solution found when resolving dependencies:
       ╰─▶ Because package-a==1.0.0 depends on package-b==1.0.0 and only the following versions of package-a are available:
               package-a==1.0.0
-              package-a>=2.0.0,<=3.0.0
+              package-a>2.0.0,<=3.0.0
```

Comment:
This is the idea from https://github.com/astral-sh/uv/pull/6071, there is no `2.0.0` so we shouldn't use `>=` here.